ydb/_topic_writer/topic_writer_asyncio_test.py

```diff
@@ -149,6 +162,31 @@ async def test_write_a_message(self, writer_and_stream: WriterWithMockedStream):
         sent_message = await writer_and_stream.stream.from_client.get()
         assert expected_message == sent_message
 
+    async def test_update_token(self, writer_and_stream: WriterWithMockedStream):
+        assert writer_and_stream.stream.from_client.empty()
+
+        await asyncio.sleep(1)
+        expected = StreamWriteMessage.FromClient(UpdateTokenRequest(token="foo-bar"))
+        got = writer_and_stream.stream.from_client.get_nowait()
+        assert expected == got, "send update token request"
+
+        await asyncio.sleep(2)
```

Comment:
Same as above + for test it is ok for set interval time smaller, then second, for example 0.01

Reply:
agree for using smaller interval, but here i want to wait some time and make sure no message was send before answer got. so nothing for wait_for_fast.